src/osd/PG.cc

```diff
@@ -2767,6 +2767,7 @@ std::pair<ghobject_t, bool> PG::do_delete_work(
     dout(20) << __func__ << " deleting " << num << " objects" << dendl;
     Context *fin = new C_DeleteMore(this, get_osdmap_epoch(), num);
     t.register_on_commit(fin);
+    osd->inc_num_deletes_in_pgs(num);
```

Comment:
Is it intentional to increase the counter the transaction is committed?

Reply:
Yes. This counter is showing total amount of deletes ever issued by OSD. And hence it never decreses.